docs/examples/cookbook/drag-and-drop.md

```diff
@@ -73,7 +73,7 @@ local function Draggable(
 
 	return New "Frame" {
 		Name = props.Name or "Draggable",
-		Parent = scope:Computed(function(use)
+		Parent = scope:Computed(function(use, scope)
```

Comment:
Scope shouldn't be provided as it is never used

Reply:
```suggestion
		Parent = scope:Computed(function(use)
```

Reply:
reverted